options/ansi/musl-generic-math/libm.h

```diff
@@ -17,7 +17,7 @@
 #include <float.h>
 #include <math.h>
 //#include <complex.h>
-#include <endian.h>
+// #include <endian.h>
```

Comment:
why was this done? why is it a comment

Reply:
endian.h is provided by the glibc option. I followed the style of complex.h and just commented it out.

Reply:
why isn't it included with it if it is? is it even required or just a false transitive include?

Reply:
We can't use that if we build ansi-only?

Reply:
This is not a public header, and the CI passes without the ``endian.h`` include, so we can assume nothing will break with this change.